services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -241,9 +241,8 @@ class DynamicSidecarSettings(BaseCustomSettings):
     DYNAMIC_SIDECAR_WAIT_FOR_CONTAINERS_TO_START: PositiveFloat = Field(
         60.0 * MINS,
         description=(
-            "After running `docker-compose up`, images might need to be pulled "
-            "before everything is started. Using default 1hour timeout to let this "
-            "operation finish."
+            "When starting container (`docker-compose up`), images might "
```

Comment:
we should discuss about these times-outs...

Reply:
Maybe let us have one final discussion about these timeouts:
- from the `dynamc-sidecar` I've removed all the timeouts which were not related with the defaults in the docker-compose API.
- I've moved all the `docker-compose` timeouts to their own module `docker_compose_utils.py` Notes:
    - `docker_compose_up` I know this could hang sometimes and would be awkward to debug. With the new long running task system we have some feedback because we know the progress so we could detect it hanging.
- the only timeouts which remain are the ones in the `director-v2`; they are all confined inside the `_public.py` module

Reply:
Regarding pint `3.`, this will be done at a later date in a different PR.